documentation/guides/package-deprecation.md

```diff
@@ -0,0 +1,42 @@
+# Deprecating a package
+
+## 1. Update `packages/{deprecate-package-name}/README.md` and `packages/{deprecate-package-name}/CHANGELOG.md`
+
+A note should be added to both of those files.
+
+Showing prominently
+
+- The last version supported.
+- Alternate to the deprecate package.
+
+## 2. Delete all files in `packages/{deprecate-package-name}`
```

Comment:
Maybe drop the `all`? Since it's not all?

Reply:
delete most? lol